docs/release-notes/v0.7.0.rst

```diff
@@ -0,0 +1,254 @@
+|release v0.7.0|_
+=================
+
+This is a minor release from ``v0.6.3`` → ``v0.7.0``.
+
+Important Notes
+---------------
+
+* Please note this release has **API breaking changes** and carefully read these
+  notes while updating your code to the ``v0.7.0`` API.
+* All backends are now fully compatible and tested with
+  `Python 3.10 <https://peps.python.org/pep-0310/>`_.
+  (PR :pr:`1809`)
+* The ``pyhf.tensorlib.poisson`` API now allows for the expected rate parameter
+  ``lam`` to be ``0`` in the case that the observed events ``n`` is ``0`` given
+  that the limit :math:`\lim_{\lambda \to 0} \,\mathrm{Pois}(n | \lambda)` is well defined.
+  (PR :pr:`1657`)
+* :func:`pyhf.readxml.parse` now supports reading of XML configurations with absolute paths.
+  To support this, ``pyhf xlm2json`` now has a ``-v/--mount`` option.
+  (PR :pr:`1909`)
+* Support for model specifications without a parameter of interest defined is added.
+  (PRs :pr:`1638`, :pr:`1636`)
+* Support for writing user custom modifiers is added.
+  (PRs :pr:`1625`, :pr:`1644`)
+* Performance in :class:`pyhf.readxml` is increased by improvements to
+  :func:`pyhf.readxml.import_root_histogram`.
+  (PR :pr:`1691`)
+* :func:`pyhf.contrib.utils.download` is now more robust to different target file types.
+  (PRs :pr:`1697`, :pr:`1704`)
+* A ``pyhf.default_backend`` has been added that is configurable through a
+  ``default`` kwarg in :func:`pyhf.set_backend`.
```

Comment:
for our work towards autodifferentiability #882

Reply:
@kratsg c.f. sentence below this line now.